build.sbt

```diff
@@ -1,12 +1,15 @@
 name := "ideprobe"
 
 val scala212 = "2.12.10"
+// don't cross compile with scala 2.13.1 for now because of issues with scala probe modules
 val scala213 = "2.13.1"
 
 skip in publish := true
 
-scalaVersion.in(ThisBuild) := scala213
-intellijBuild.in(ThisBuild) := "202.6397.20-EAP-SNAPSHOT"
+scalaVersion.in(ThisBuild) := scala212
+// -EAP-SNAPSHOT suffix results in incorrect url for the intellij scala plugin
+// it is appended automatically for intellij sdk dependency url and manually for the BuildInfo
+intellijBuild.in(ThisBuild) := "202.6397.20"
```

Comment:
we could just bump to stable 2020.2.1 (still needs a number like 202.xxxx.xx)

Reply:
done